checker/tests/test_isAnDO.py

```diff
@@ -1,5 +1,5 @@
 import unittest
-import AnDOChecker as ando
+import AnDO_engine as andoE
```

Comment:
is there a reason for this change from "ando" to "andoE"?

Reply:
Yes because it uses the AnDO_engine not the AnDOChecker it makes more sense to me

Reply:
ok!